dmr/src/main/java/org/jboss/dmr/client/dispatch/impl/DMRHandler.java

```diff
@@ -270,7 +270,7 @@ public void onError(Request request, Throwable e)
             request = requestBuilder.send();
             trace(Type.SEND, id, operation);
         }
-        catch (RequestException e)
+        catch (Throwable e)
```

Comment:
Also suggesting this, as it would allow for unexpected errors (like the one being solved here) to be reported by red error message dialog, instead of being silently swallowed.